LibreNMS/Authentication/LdapAuthorizer.php

```diff
@@ -152,6 +152,9 @@ public function getUserlist()
             }
 
             $filter = '(' . Config::get('auth_ldap_prefix') . '*)';
+            if (Config::get('auth_ldap_Userlist_filter') != null) {
```

Comment:
Could you rewrite the option name with low cap instead of "U" ?

Reply:
Yes, no problem, I do It.